app/Http/Controllers/Tquery/AdminUserTqueryController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Tquery;
+
+use App\Http\Controllers\ApiController;
+use App\Http\Permissions\Permission;
+use App\Http\Permissions\PermissionDescribe;
+use App\Services\Tquery\OpenApiGet;
+use Illuminate\Http\JsonResponse;
+use stdClass;
+
+class AdminUserTqueryController extends ApiController
+{
+    protected function initPermissions(): void
+    {
+        $this->permissionOneOf(Permission::globalAdmin);
+    }
+
+    #[OpenApiGet(
+        path: '/api/v1/admin/user/tquery',
+        permissions: new PermissionDescribe(Permission::globalAdmin),
+        summary: 'All users tquery',
+        tag: 'Admin',
+    )]
+    public function get(): JsonResponse
+    {
+        return new JsonResponse([
+            'data' => [
+                'columns' => [
+                    ['name' => 'id', 'type' => 'uuid'],
+                    ['name' => 'name', 'type' => 'string'],
+                    ['name' => 'email', 'type' => 'string'],
+                    //todo: 'name' => 'lastLoginFacilityId', 'type' => 'facility' (?)
+                    ['name' => 'lastLoginFacility', 'type' => 'string'],
+                    ['name' => 'passwordExpireAt', 'type' => 'datetime'],
+                    ['name' => 'hasPassword', 'type' => 'bool'],
+                    ['name' => 'createdAt', 'type' => 'datetime'],
+                    ['name' => 'updatedAt', 'type' => 'datetime'],
+                    ['name' => 'hasEmailVerified', 'type' => 'bool'],
+                    //todo: 'type' => 'user'
+                    ['name' => 'createdBy', 'type' => 'user'],
+                    ['name' => 'hasGlobalAdmin', 'type' => 'bool'],
+                    ['name' => 'members', 'type' => 'text'],
+                ],
+                'customFilters' => new stdClass(),
+            ],
```

Comment:
Możesz też dodać:
```
suggestedColumns:[name,email,...],
suggestedSort:[{type: "column", column: "name", dir: "asc"}],
```

Reply:
w sumie to samo tquery może zasilać np tabelę i kalendarz, i to chyba jednak fe powinien takie rzeczy wiedzieć

Reply:
Możemy pewnie założyć że klient powinien wiedzieć. Klient i tak już będzie miał sporo customowego kodu na konkretne kolumny, to by tylko było trochę dodatkowego customowego kodu.